h/views/activity.py

```diff
@@ -35,10 +35,13 @@ class SearchController(object):
 
     def __init__(self, request):
         self.request = request
+        # Cache a copy of the extracted query params for the child controllers to use if needed.
+        self.parsed_query_params = query.extract(self.request)
```

Comment:
Why is this necessary?

Reply:
And this is necessary because search eats the params.